docs-en/chapter_preface/summary.md

```diff
@@ -1,8 +1,7 @@
 # Summary
 
-- The main audience of this book is beginners in algorithm. If you already have some basic knowledge, this book can help you systematically review your algorithm knowledge, and the source code in this book can also be used as a "brushing tool library".
+- The main audience of this book is beginners in algorithm. If you already have some basic knowledge, this book can help you systematically review your algorithm knowledge, and the source code in this book can also be used as a "Coding Toolkit".
 - The book consists of three main sections, Complexity Analysis, Data Structures, and Algorithms, covering most of the topics in the field.
-- For newcomers to algorithms, it is crucial to read an introductory book in the beginning stages to take the road less traveled.
-- Animations and graphs within the book are usually used to introduce key points and difficult knowledge. These should be given more attention when reading the book.
+- For newcomers to algorithms, it is crucial to read an introductory book in the beginning stages to avoid many detours or common pitfalls.- Animations and graphs within the book are usually used to introduce key points and difficult knowledge. These should be given more attention when reading the book.
 - Practice is the best way to learn programming. It is highly recommended that you run the source code and type in the code yourself.
```

Comment:
please correct the formats, there should be a line break before "-Animations"

Reply:
fix it, ty